opi/__init__.py

```diff
@@ -265,9 +329,13 @@ def ask_number(min_num, max_num, question="Choose a number (0 to quit):"):
 	else:
 		return ask_number(question, min_num, max_num)
 
+
 def ask_keep_repo(repo):
 	if not ask_yes_or_no('Do you want to keep the repo "%s"?' % repo, 'y'):
-		subprocess.call(['sudo', 'zypper', 'rr', repo])
```

Comment:
Beware that this might get called not with a repo alias (=filename) but with a repo name.

Reply:
I'll work on these changes, but especially this gives me quite a headache and I am not yet sure how I can do this

Reply:
Maybe have a look where this is actually called - I tried to make my latest changes use the repo alias (=filename) but I'm not 100% sure if it is called everywhere with the alias. Maybe it would be easier to make sure that it is only called with the alias.

Reply:
Alternatively this could be handled by parsing all repo files and comparing the names.

Reply:
I just had a look and it seems that there is currently no remaining call with the repo name.
The only call that was using the repo name instead of the alias was moved to the alias in https://github.com/openSUSE/opi/commit/50c43d064468c87c34713ecb6e4f6ca8550bee89